docs/requirements.txt

```diff
@@ -1,2 +1,2 @@
-mkdocs==1.3.0
-markdown-include==0.6.0
+mkdocs==1.6.1
```

Comment:
Do we need to pin griffe too or odes this project not use it? Also, does pyproject.toml need to be updated for this project?

Reply:
It's old docs, before the revamp, so not using mkdocstrings. pyproject and the lockfile are good.